server/odc-service/src/main/java/com/oceanbase/odc/service/config/SystemConfigService.java

```diff
@@ -57,6 +66,10 @@ public class SystemConfigService {
     @Autowired
     private List<SystemConfigRefreshMatcher> systemConfigRefreshMatchers;
 
+    private final LoadingCache<String, List<Configuration>> configCache = Caffeine.newBuilder().maximumSize(1)
```

Comment:
tag service should responsible for performance, so the cache may invovled in tag service

Reply:
`configCache` works for the tag service as well.